src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -62,4 +62,13 @@ object FireCloudConfig {
     def importEntitiesPathFromWorkspace(namespace: String, name: String) = authUrl + importEntitiesPath.format(namespace, name)
   }
 
+  object Thurloe {
+    private val profile = config.getConfig("userprofile")
+    lazy val baseUrl = sys.env.get("THURLOE_URL_ROOT").get
+    lazy val setKey = profile.getString("setKey")
```

Comment:
This is the only part of this PR I don't like, but I would not change it (so your code remains consistent with the rest of the codebase). I recommend we do away with this pattern via https://github.com/broadinstitute/firecloud-orchestration/pull/119